package.json

```diff
@@ -5,10 +5,13 @@
   "private": true,
   "license": "MPLv2",
   "scripts": {
+    "start:a": "npm run build && web-ext run -s dist/extension-a-$npm_package_version",
+    "start:b": "npm run build && web-ext run -s dist/extension-b-$npm_package_version",
+    "start:c": "npm run build && web-ext run -s dist/extension-c-$npm_package_version",
     "build": "node build.js",
     "clean": "rimraf dist web-ext-artifacts",
     "lint": "eslint . --ext jsm,js,json",
-    "format": "npm-run-all format:*",
+    "format": "npm-run-all -s -n format:prettier format:eslint",
```

Comment:
Can this use the `--long-form` version of the arguments, so that it is is clearer what it does?